benchmarks/parquet.rs

```diff
@@ -84,13 +86,132 @@ async fn main() -> Result<()> {
     }
 
     let test_file = gen_data(path, opt.scale_factor, props_builder.build())?;
-
-    run_benchmarks(opt, &test_file).await?;
-
+    println!("running filter benchmarks");
+    run_filter_benchmarks(opt.clone(), &test_file).await?;
+    println!("running sort benchmarks");
```

Comment:
what would you think about adding some way to pick which benchmark to run here. Something like

Run the sort benchmark  like
```shell
cargo run --release --bin parquet -- sort --path ./data --scale-factor 1.0
```

Run the filter benchmark  like
```shell
cargo run --release --bin parquet -- filter --path ./data --scale-factor 1.0
```